app/v3/service/internal/types/msgs.go

```diff
@@ -958,6 +963,10 @@ func ensureServiceDefLength(msg MsgDefineService) sdk.Error {
 	}
 
 	for i, tag := range msg.Tags {
+		if len(tag) < MinTagLength {
```

Comment:
why not
``` 
len(tag) == 0
```

Reply:
Add a MinTagLength which can be more flexible？

Reply:
> Add a MinTagLength which can be more flexible？

but it is equal to 1, doesn't make any sense.